SYCL/Sampler/unnormalized-clamp-linear-float.cpp

```diff
@@ -10,6 +10,9 @@
 // On Windows, LevelZero returns wrong result for outside right pixel when using
 // unorm_int8 data.
 
+// #204 Disable unstable tests
+// REQUIRES: TEMPORARY_DISABLED
```

Comment:
Probably it should be disabled only for one configuration: Windows + Level Zero.

Reply:
Changed it as Chris suggested.